ghedesigner/search_routines.py

```diff
@@ -269,9 +269,24 @@ def search(self):
         keys = list(self.calculated_temperatures.keys())
         values = list(self.calculated_temperatures.values())
 
+        # theoretically, the biggest negative value should be the field that is just undersized
         negative_excess_values = [v for v in values if v <= 0.0]
-
         excess_of_interest = max(negative_excess_values)
+
+        # but some conditions don't yield this result
+        # adding a check here to ensure we pick the smallest field with
+        # negative excess temperature
+        num_bh = [len(self.coordinates_domain[x]) for x in keys]
+        sorted_num_bh, sorted_values = (list(t) for t in zip(*sorted(zip(num_bh, values))))
+        for nbh, val in zip(sorted_num_bh, sorted_values):
+            if val < 0:
+                if excess_of_interest != val:
+                    print('Loads resulted in odd behavior requiring the selected field configuration \n'
+                          'to be reset to the smallest field with negative excess temperature. \n'
+                          'Please forward the inputs to the developers for investigation.')
+                excess_of_interest = val
+                break
```

Comment:
Here's the primary reason for this PR. The user's input file had a strange set of load combinations that caused the biggest field to have the maximum negative excess temperature. This checks to verify that the field selected through the above logic is also the field that has negative excess temperature and is the smallest field of those with negative excess temp.